.github/workflows/publish.js.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will do a clean install of node dependencies, build the source code, run tests, and publishes to github.
+
+name: Test and publish to GitHub
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  build_push_test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: npm install -g yarn
+      - run: yarn install
+      - run: yarn build
+      - name: Upload
+        uses: svenstaro/upload-release-action@v2
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: build.zip
```

Comment:
Maybe name the file 'rf-coverage-maps.zip'?

Reply:
This is the local file name. Asset_name will be the name on the upload. Will update.
Docs: https://github.com/svenstaro/upload-release-action/releases